src/main/java/org/asamk/signal/manager/Manager.java

```diff
@@ -810,10 +810,12 @@ public String getGroupName(byte[] groupId) {
     /**
      * Change the expiration timer for a contact
      */
-    public void setExpirationTimer(SignalServiceAddress address, int messageExpirationTimer) {
```

Comment:
please keep the method with `SignalServiceAddress` parameter and just add a new method (with the same name) that takes a `String` and does the `canonicalizeAndResolveSignalServiceAddress` conversion.

Reply:
edited, you can review the changes